integration/heapster_api_test.go

```diff
@@ -630,10 +625,10 @@ func apiTest(kubeVersion string) error {
 	if err := createAll(fm, ns, &svc, &rc); err != nil {
 		return err
 	}
-	if err := fm.WaitUntilPodRunning(ns, rc.Spec.Template.Labels, time.Minute); err != nil {
+	if err := fm.WaitUntilPodRunning(ns, rc.Spec.Template.Labels, 5*time.Minute); err != nil {
```

Comment:
Just to check if this helps. There were multiple runs (only in Jenkins) where service could not find his endpoints within the previous timeouts.